src/operator/context/account.tsx

```diff
@@ -138,23 +153,42 @@ export const AccountProvider: FC<Props> = React.memo(({children}) => {
     }
   }, [dispatch, history, accountID])
 
+  const handleReactivateAccount = useCallback(async () => {
+    try {
+      setReactivateStatus(RemoteDataState.Loading)
```

Comment:
Is `reactivateStatus` being used? This pattern gets used in our Redux state-managed components to ensure that in `useEffect` hooks, we only trigger API calls once on page load, not when loading or when there's an error. But I don't see anything in these components that relies on this status.

Reply:
I started to remove it, but I instead kept it to disable the reactivate button once they click it. Technically, the operation is idempotent, but it's probably better if they're not smashing it, since it can be a slow operation.